src/main/java/seedu/contax/ui/MainWindow.java

```diff
@@ -115,6 +115,8 @@ private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
      */
     void fillInnerParts() {
         personListPanel = new PersonListPanel(logic.getFilteredPersonList());
+        System.out.println(logic.getAppointmentList());
```

Comment:
was this intentional to list the appointments in the console?

Reply:
Sorry! You are right, fixed in commit 61c2e99.